bgls/simulator_test.py

```diff
@@ -278,3 +280,60 @@ def test_remains_clifford():
     sim_results_stab = sim_act_on_stab.run(clifford_circuit, repetitions=100)
 
     assert sim_results == sim_results_stab
+
+
+def test_mps_results_match_state_vec():
+    """Test sampled bitstrings are same when using a matrix product state
+    simulator and a state vector simulator.
+    """
+    try:
+        import quimb.tensor as qtn
+        import cirq.contrib.quimb.mps_simulator
+    except ImportError:
+        return "quimb not installed, mps simulation not possible"
```

Comment:
Looking earlier on this thread it used to be in `requirements.txt`, and we removed it since it wasn't a core requirement per se. Does putting it in `dev_requirements.txt` address this?